.travis.yml

```diff
@@ -11,6 +11,8 @@ before_install:
     - wget http://bit.ly/miniconda -O miniconda.sh
     - bash miniconda.sh -b -p $HOME/miniconda
     - export PATH="$HOME/miniconda/bin:$PATH"
+    - conda update --yes --all
```

Comment:
Please remove this line. There is no need to *update all here* and the CI will run faster without it. (That line was removed in PR #87.)